crates/redislog/src/lib.rs

```diff
@@ -258,17 +258,27 @@ impl Logger {
 
 impl slog::Drain for Logger {
     type Ok = ();
-    type Err = self::Error;
```

Comment:
Since I believe the policy about dropping or panicking should still sit with the user of the crate I'd like us to try this approach first: https://github.com/bolcom/unFTP/pull/95

Reply:
I've reworked this PR based on your idea, but I also added sending the `Error` of the drains to stderr.
I've also changed from `FullFormat` to `CompactFormat` to keep things async.